lib/datadog/tracing/contrib/karafka/monitor.rb

```diff
@@ -15,6 +15,8 @@ module Monitor
             worker.processed
           ].freeze
 
+          EMPTY_HASH = {}.freeze
+
           def instrument(event_id, payload = EMPTY_HASH, &block)
```

Comment:
I would also recommend to rename it into `EMPTY_PAYLOAD` to avoid using structure in the name to have some encapsulation of the implementation details.

```suggestion
          EMPTY_PAYLOAD = {}.freeze

          def instrument(event_id, payload = EMPTY_PAYLOAD, &block)
```